### DISCUSSION TITLE
How to use pyannote-audio for diarization ?

### DISCUSSION THREAD

**Discussion by mkr00** [pyannote/pyannote-audio#889] [Q&A]
Hello ,
  Could anyone tell me how to use pyannote-audio for diarization, please ?. If I want to use my wave file and I want to get the result of diarization in the rttm file. 

BR
mkr00

**Comment by Yagna24** [comment]
To load the diarization pipeline use this  : 

`pipeline = torch.hub.load('pyannote/pyannote-audio', 'dia_ami') 
`
Your .wav file should be passed like this : 

`AUDIO = {'uri': 'ES2004a.Mix-Headset', 'audio': '/content/ES2004a.Mix-Headset.wav'}`

To run the pipeline :  

`diarized = pipeline(AUDIO)`

Results : 

`segments = diarized.for_json()["content"]`    -->  to get the timestamps and labels 

`diarized --> To view the plot `

You could then convert the output to rttm format
